webapp-shadcn/src/app/(dashboard)/projects/[projectId]/page.tsx

```
@@ -0,0 +1,964 @@
+        </TabsContent>
+
+        {/* Integrations Tab - Figma + AI */}
+        <TabsContent value="integrations" className="space-y-4">
+          <div className="grid gap-6 md:grid-cols-2">
+            {/* Figma Integration */}
+            <Card>
+              <CardHeader>
+                <div className="flex items-center gap-3">
+                  <div className="p-2 bg-[#F24E1E]/10 rounded-lg">
+                    <svg className="h-6 w-6" viewBox="0 0 38 57" fill="none">
+                      <path d="M19 28.5C19 23.2533 23.2533 19 28.5 19C33.7467 19 38 23.2533 38 28.5C38 33.7467 33.7467 38 28.5 38C23.2533 38 19 33.7467 19 28.5Z" fill="#1ABCFE"/>
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)